tests/testit/test-cran.R

```diff
@@ -0,0 +1,43 @@
+library(testit)
+
+assert("cran_package_dates() needs the name of a maintainer", {
+  has_error(cran_pkg_dates(maintainer = NULL))
+})
+
+assert("cran_package_dates() returns the good class", {
+  x <- cran_pkg_dates()
+  lapply(x, function(y) {
+    class(y) %==% "Date"
+  })
+
+  # several dates per package, hence nested lapply
+  x <- cran_pkg_dates(full = TRUE)
+  lapply(x, function(y) {
+    lapply(y, function(z) {
+      class(z) %==% "Date"
+    })
+  })
+
+})
```

Comment:
Overall I feel this particular function and the `cran_updatable()` below are not suitable for testing. They are mainly for my own use. I can't think of any meaningful tests for them.